src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixAbstractMethodsStep.cs

```diff
@@ -62,7 +62,7 @@ void CheckAppDomainUsageUnconditional (AssemblyDefinition assembly, Action<strin
 
 			foreach (var mr in assembly.MainModule.GetMemberReferences ()) {
 				if (mr.ToString ().Contains ("System.AppDomain System.AppDomain::CreateDomain")) {
-					warn ($"warning XA2000: Use of AppDomain.CreateDomain() detected in assembly: {assembly}. .NET 5 will only support a single AppDomain, so this API will no longer be available in Xamarin.Android once .NET 5 is released.");
+					warn (string.Format ("warning XA2000: " + Xamarin.Android.Tasks.Properties.Resources.XA2000, assembly));
```

Comment:
It occurred to me after some consideration that "warning" might need to be localized too. I'll have to do some quick investigation on that.

Reply:
if you localise that is might perhaps break the MSBuild Error/Warning parser, which is what this depends on.

Reply:
I confirmed that this is all set. The `warning XA2000` prefix should *not* be localized. I ran MSBuild in a different locale and checked the `warning` prefix produced by `LogCodedWarning()`. The prefix is *not* localized even when the rest of the message is localized.